app/jobs/populate_register_data_in_db_job.rb

```diff
@@ -10,6 +10,7 @@ def perform(register)
     begin
     # Initialize client and download / update data
       register_client = RegistersClientWrapper.registers_client.get_register(register.name.parameterize, register.register_phase.downcase, PostgresDataStore.new(register))
+      register_url = register_client.instance_variable_get(:@register_url)
```

Comment:
I think this should be a public method on the client library but that needs an upstream change, so this is OK for now.

Reply:
👍